frontend/messages/pl.json

```diff
@@ -0,0 +1,84 @@
+	"settingsWindShowNumericalValuesLegend": "Pokaż prędkość wiatru jako wartość liczbową obok strzałki wiatru",
+	"settingsTime": "Czas",
+	"settingsTimeExplanation": "Czas może być wyświetlany zgodnie z Twoją strefą czasową lub w UTC.",
+	"settingsTimeUserTimezone": "Pokaż czas, używając mojej strefy czasowej",
+	"settingsTimeUserTimezoneLegend": "Pokaż czas, używając mojej strefy czasowej zamiast UTC",
+	"settingsTimeUTC": "Pokaż czas według UTC",
+	"settingsTimeUTCLegend": "Pokaż czas UTC zamiast mojej strefy czasowej",
+	"locationSummary": "Streszczenie",
+	"locationSummaryLegend": "Podsumowanie prognozy dla tej lokalizacji",
+	"locationMeteogram": "Meteogram",
+	"locationMeteogramLegend": "Meteogram dla tej lokalizacji",
+	"locationSounding": "Sondaż",
+	"locationSoundingLegend": "Sondaż dla tego czasu i miejsca",
+	"period24HoursBefore": "24 godziny wcześniej",
+	"periodPrevious": "Poprzedni okres prognozy",
+	"periodNext": "Następny okres prognozy",
+	"period24HoursAfter": "24 godziny później",
+	"layerCloudsAndRain": "Chmury i deszcz",
+	"layerCloudsAndRainLegend": "chmury i deszcz",
+	"layerCumulusDepth": "Chmury Cumulus",
+	"layerCumulusDepthLegend": "głębokość chmur cumulusowych",
+	"layerThermalVelocity": "Prędkość termiki",
+	"layerThermalVelocityLegend": "prędkość prądu termicznego",
+	"layerThQ": "Jakość termiki",
+	"layerThQLegend": "jakość termiki",
+	"layerWindSurface": "Powierzchnia",
+	"layerWindSurfaceLegend": "prędkość i kierunek wiatru na ziemi",
+	"layerWind300MAGL": "300 m nad gruntem",
+	"layerWind300MAGLLegend": "prędkość i kierunek wiatru na wysokości 300 m nad poziomem gruntu",
+	"layerWind2000AMSL": "2000 m n.p.m",
+	"layerWind2000AMSLLegend": "prędkość i kierunek wiatru na wysokości 2000 m nad poziomem morza",
+	"layerWind3000AMSL": "3000 m n.p.m",
+	"layerWind3000AMSLLegend": "prędkość i kierunek wiatru na wysokości 3000 m nad poziomem morza",
+	"layerWind4000AMSL": "4000 m n.p.m",
+	"layerWind4000AMSLLegend": "prędkość i kierunek wiatru na wysokości 4000 m nad poziomem morza",
+	"summaryTotalCloudCover": "Całkowite zachmurzenie",
+	"summaryRainfall": "Opady",
+	"summaryCumuliDepth": "Wysokość cumulusów",
+	"summaryThermalVelocity": "Prędkość termiki",
+	"summaryWeekOverview": "Przegląd tygodnia",
+	"summaryDayOverview": "Przegląd dnia",
+	"summaryThQ": "Jakość termiki",
+	"summaryWindSpeed": "Prędkość wiatru",
+	"zoneAlpsOverview": "Przegląd Alp (6 km)",
+	"zoneCentralAlps": "Alpy Środkowe (2 km)",
+	"zoneSouthernAlps": "Alpy Południowe (2 km)",
+	"zoneEasternAlps": "Alpy Wschodnie (2 km)",
+	"zoneEurope": "Europa",
+	"zoneAustralia": "Australia",
+	"zoneHimalaya": "Himalaje",
+	"zoneNorthAmerica": "Ameryka północna",
+	"zoneSouthAfrica": "Afryka Południowa",
+	"zoneSouthAmerica": "Ameryka Południowa",
+	"helpIntro1": "Soaringmeteo to bezpłatna witryna z prognozą pogody stworzona przez pasjonatów pilotów. Wesprzesz?",
+	"helpIntro2": "Darowizna",
+	"helpIntro3": "Pomożesz nam pokryć koszty.",
```

Comment:
Note that this sentence is broken down into three different messages for the needs of formatting. You can see the result in English here:

![image](https://github.com/soaringmeteo/soaringmeteo/assets/332812/ff8f5b59-c1c9-4c61-aa36-5ddcb1461085)

I can not review the Polish version, but could you please confirm that the result will be similar?

Reply:
Hi Julien,

I have noticed few other cases of messages broken into parts. I translated with that in mind, I envisioned scrolling or breaking lines or "paging". I see it was "hyperlinking". In all cases my approach works well - english "donation" is pl: "Darowizna". I formulated more as "phrases" rather than continuous sentence, hence capital letters. I believe it is OK

Reply:
Thank you for the explanation. You know better than me how to translate idiomatically in Polish :)